dsp-shared/src/main/scala/dsp/valueobjects/Iri.scala

```diff
@@ -99,29 +104,30 @@ object Iri {
    */
   sealed abstract case class ProjectIri private (value: String) extends Iri
   object ProjectIri { self =>
-    def make(value: String): Validation[Throwable, ProjectIri] =
+    def make(value: String): Validation[ValidationException, ProjectIri] =
       if (value.isEmpty) {
-        Validation.fail(BadRequestException(IriErrorMessages.ProjectIriMissing))
+        Validation.fail(ValidationException(IriErrorMessages.ProjectIriMissing))
       } else {
         val isUuid: Boolean = V2UuidValidation.hasUuidLength(value.split("/").last)
 
         if (!V2IriValidation.isKnoraProjectIriStr(value)) {
-          Validation.fail(BadRequestException(IriErrorMessages.ProjectIriInvalid))
+          Validation.fail(ValidationException(IriErrorMessages.ProjectIriInvalid))
         } else if (isUuid && !V2UuidValidation.isUuidVersion4Or5(value)) {
-          Validation.fail(BadRequestException(IriErrorMessages.UuidVersionInvalid))
+          Validation.fail(ValidationException(IriErrorMessages.UuidVersionInvalid))
         } else {
-          val validatedValue = Validation(
+          val eitherValue = Try(
             V2IriValidation.validateAndEscapeProjectIri(
               value,
-              throw BadRequestException(IriErrorMessages.ProjectIriInvalid)
+              throw ValidationException(IriErrorMessages.ProjectIriInvalid)
```

Comment:
Could you add a test to cover this case?

Reply:
Not really, from what I see - the check here is actually redundant. I just didn't want to have throwing code that's not wrapped in a Validation.

Reply:
Also, the validation used here is located in the `V2` package where we make legacy code available for the new code base. But this validation should be reworked anyways, once we have the domain model with all definitions of validity